posts/2023-10-11-devoxx-be-2023.adoc

```diff
@@ -0,0 +1,79 @@
+= Devoxx BE 2023
+TODO author
```

Comment:
Btw it looks like this can either be a single author or none, so I think you might need to pick on or leave it blank.

Reply:
Ooooo nevermind! https://docs.asciidoctor.org/asciidoc/latest/document/author-information/#multiple-author-attributes. So it's outlined how to do it here and `authors` is in https://github.com/lunatech-labs/lunatech-blog-engine/blob/5483e023a2934d50f4e92d8268616779f76bf3f5/app/models/Post.scala#L28, but not sure if it's used or not elsewhere. I see author just grabs the first one. Either way I think you can put all three comma separated in here.

Reply:
Done!